src/Akka.Persistence.Sql.Linq2Db/Config/MetadataTableColumnNames.cs

```diff
@@ -51,26 +52,5 @@ public override int GetHashCode()
         {
             return HashCode.Combine(PersistenceId, SequenceNumber);
         }
-
-        
-        
-        public static readonly string FallBack = @"tables.journal{
```

Comment:
Baking hardwired HOCON config is a bad idea, this creates 4 copies of the same HOCON config in 4 different place